asdf/tests/test_search.py

```diff
@@ -189,3 +189,17 @@ def test_recursive_tree():
     result.replace("zap")
     assert af["foo"]["bar"] == "zap"
     assert af["foo"]["nested"]["bar"] == "zap"
+
+
+def test_search():
+    tree = dict(foo=42, bar="hello", baz=np.arange(20))
+    af = asdf.AsdfFile(tree)
```

Comment:
```suggestion
    af = AsdfFile(tree)
```

I think this will fix the broken tests

Reply:
Already done.